src/loan_form/loan_entry/LoanEntryField.jsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react'
 
-export default function LoanEntryField({ field_id, name, value, label, on_change }) {
+export default function LoanEntryField({ fieldClass, name, value, label, initial, onChange }) {
   return (
     <div>
       <label>{label}</label>
       <input
         type="text"
-        id={field_id}
+        className={`loan-entry-${fieldClass}`}
         name={name}
-        placeholder={value}
-        onChange={on_change} />
+        placeholder={initial}
+        value={value}
```

Comment:
if these are immediately rendered with a value other than an empty string (true for all but title), the placeholder doesn't show unless the user deletes what's already in the field. so: do we need placeholders? in terms of usability strings of this length would be better served as helper text or even tooltips, anyways